actors/miner/src/lib.rs

```diff
@@ -878,6 +880,7 @@ impl Actor {
         }
         burn_funds(rt, aggregate_fee)?;
         state.check_balance_invariants(&rt.current_balance()).map_err(balance_invariants_broken)?;
+
```

Comment:
Nit: unnecessary blank line. Some others similar below.

Reply:
This is done.